gatsby-node.js

```diff
@@ -100,12 +100,6 @@ exports.onPostBuild = async function (
 
     const objects = await transformer(result);
 
-    if (objects.length > 0 && !objects[0].objectID) {
```

Comment:
this error then would need to be put back (in case no object id or id)

Reply:
why don't we accept this situation. There is no restriction on the algolia side. 🤔

Reply:
if you don't have an objectID, you won't be able to find those records again for updating later, and will be forced to fall back to a full reindex, which is more operations

Reply:
https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/#unique-identifier---objectid 

I know this situation, but as I said, there is no restriction. But it's not a problem if we make it mandatory 👍